cpp/daal/src/algorithms/dtrees/forest/df_train_dense_default_impl.i

```diff
@@ -633,6 +729,148 @@ typename DataHelper::NodeType::Base * TrainBatchTaskBase<algorithmFPType, DataHe
     return makeLeaf(_aSample.get() + iStart, n, curImpurity, nClasses);
 }
 
+// Breadth-first search
```

Comment:
Why do you need breadth-first instead of best-first?

Reply:
it was intermediate implementation, helped to create best-first